tests/contrib/django_util/settings.py

```diff
@@ -12,8 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+"""This module provides a base Django settings module used by the rest of the
+tests."""
+
 import os
 
+INSTALLED_APPS = [
+    'django.contrib.admin',
+    'django.contrib.auth',
+    'django.contrib.contenttypes',
+    'django.contrib.sessions',
+    'django.contrib.messages',
+    'django.contrib.staticfiles',
+    'oauth2client.contrib.django_util',
+    'tests.contrib.django_util.apps.DjangoOrmTestApp'
```

Comment:
For multi-line sequence literals with one element per line, always include the optional comma after the last element.